devlib/utils/misc.py

```diff
@@ -590,6 +591,31 @@ def __str__(self):
         return message.format(self.filepath, self.lineno, self.message)
 
 
+def load_struct_from_yaml(filepath):
```

Comment:
For now we don't want the testing infrastructure to be a public API, so this needs a non-public name starting with a single underscore (https://peps.python.org/pep-0008/)

Reply:
`load_struct_from_yaml()` is not for testing purposes only.
It's just moved from WA to devlib: https://github.com/ARM-software/workload-automation/pull/1248